config/settings.yml

```diff
@@ -20,8 +20,11 @@ govuk_notify:
 # When set to true, any capybara tests will run chrome normally rather than in headless mode.
 show_browser_during_tests: false
 
-# When set to true, All pages will render 'Maintenance mode' message
-maintenance_mode_enabled: false
+maintenance_mode:
+  # When set to true, All pages will render 'Maintenance mode' message
+  enabled: false
+  # List of IP addresses which will bypass the maintenance mode message
```

Comment:
Might be worth mentioning its a comma separated list of IPs?

Reply:
I've addressed this comment in another PR so we dont have to rebase and ask for another approval